### PR TITLE
Update GenRepoInfo configs and remove last of the File references

### PR DESCRIPTION
**Pull request by loosebazooka**
part of #723

### CODE REVIEW

File: src/main/java/com/google/cloud/tools/appengine/cloudsdk/internal/args/Args.java [hunk 1]
```diff
@@ -95,26 +94,14 @@
...
-    if (file != null) {
-      return path(name, file.toPath());
-    }
-    return Collections.emptyList();
-  }
-
   /**
    * Produces the flag form of a path object, using {@link Path#toString()}.
    *
    * @return {@code [--name, path.toString()]} or {@code []} if path is null or not set.
```
Comment (etanshaul) [inline src/main/java/com/google/cloud/tools/appengine/cloudsdk/internal/args/Args.java]: javadoc updated too to match changes?

File: src/main/java/com/google/cloud/tools/appengine/cloudsdk/internal/args/Args.java [hunk 2]
```diff
@@ -95,26 +94,14 @@
...
    */
   static List<String> path(String name, @Nullable Path path) {
     if (path != null && !path.toString().isEmpty()) {
-      return Arrays.asList("--" + name, path.toString());
+      return Arrays.asList("--" + name, path.toAbsolutePath().toString());
```
Comment (etanshaul) [inline src/main/java/com/google/cloud/tools/appengine/cloudsdk/internal/args/Args.java]: could this cause any behavior changes for the callers or is it the same thing?

File: src/main/java/com/google/cloud/tools/appengine/cloudsdk/internal/args/Args.java (hunk 1 above)
Comment (loosebazooka) [reply src/main/java/com/google/cloud/tools/appengine/cloudsdk/internal/args/Args.java]: ah, yeah keep missing those.

File: src/main/java/com/google/cloud/tools/appengine/cloudsdk/internal/args/Args.java (hunk 2 above)
Comment (loosebazooka) [reply src/main/java/com/google/cloud/tools/appengine/cloudsdk/internal/args/Args.java]: I looked through it, and it seemed fine. But I'll go through and check again. There actually weren't too many callers.

Comment (loosebazooka) [reply src/main/java/com/google/cloud/tools/appengine/cloudsdk/internal/args/Args.java]: so it looks like this should be okay, the callers (in tests) were all provided with absolute paths. And all commands that execute with this parameter should be taking absolute paths anyway.

Comment (loosebazooka) [reply src/main/java/com/google/cloud/tools/appengine/cloudsdk/internal/args/Args.java]: I've pushed an update to the tests to make it explicit.

**Pull request merged**